docs/pages/components/figma.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+
+import { Figma } from '../../../packages/mdx-embed/src/components/figma';
+
+<Meta title="Components/Figma" component={Figma} />
+
+# CodePen
+
+Display a Figma file by including the component in your `.mdx`. The `title` and `url` props are required.
+
+<ArgsTable of={Figma} />
+
+## Usage
+
+<Canvas>
+  <Story name="usage">
+    <Figma title="Boop" url="https://www.figma.com/file/LKQ4FJ4bTnCSjedbRpk931/Sample-File" />
```

Comment:
I might need to investigate this but... generally we don't include the url in the prop: 

`https://www.figma.com/` could be added inside the component then the prop could be `link` or something. I'll have to check on some of the other components and provide you some examples because it's not a hard and fast rule from what i remember.

Reply:
Here's some examples of urls hardcoded in the component
- [buzzsprout](https://github.com/PaulieScanlon/mdx-embed/blob/main/packages/mdx-embed/src/components/buzzsprout/buzzsprout.tsx#L29)
- [codepen](https://github.com/PaulieScanlon/mdx-embed/blob/main/packages/mdx-embed/src/components/codepen/codepen.tsx#L36)
- [spotify](https://github.com/PaulieScanlon/mdx-embed/blob/main/packages/mdx-embed/src/components/spotify/spotify.tsx#L22)

Reply:
Thats fair, i was following the official docs on how to embed. If we choose to go the route of appending the url id then users will have to copy out that part instead of the whole url. Bit of a pain in the arse if you ask me. But its your package, i dont knwo what yo have done eslewhere either

Reply:
Right, buzzsprout has the regex test too. I just saw figma has one you can use too. Ive had a few wines. Will add that for sure. about the url, its up to you. Do you want users to copy the specific id from the url? If so then i will change it

Reply:
Yeah this url thing came up before, it's a bit of a pain for this and perhaps a few others but there are lots that are easier if you don't pass the URL.

The trouble is changing this on any of the current components will be a breaking change and so the best approach for now is keep this consistent with all the others and when i can be arsed i'll have to comb over everything and see if there's a common / better way then release a v1 or maybe v2